components/molecule/modal/src/index.js

```diff
@@ -125,11 +120,8 @@ const MoleculeModal = ({
     const dialogClassName = cx(suitClass({element: 'dialog'}), {
       [suitClass({element: 'dialog--full'})]: fitWindow,
       [suitClass({element: 'dialog--out'})]: isClosing,
-      [suitClass({element: 'dialog--fit'})]: fitContent
-    })
-
-    const contentClassName = cx(suitClass({element: 'content'}), {
-      [suitClass({element: 'content--without-indentation'})]: withoutIndentation
+      [suitClass({element: 'dialog--fit'})]: fitContent,
+      [suitClass({element: `dialog--${size}`})]: !!size
```

Comment:
[Subjective opinion] Could we use a descriptive way? `Boolean(size)`.
It is more semantic